.github/workflows/main.yml

```diff
@@ -115,6 +115,8 @@ jobs:
     strategy:
       matrix:
         pair:
+          - erlang: 22.x
+            elixir: 1.10
```

Comment:
Use a string here, otherwise it will be `1.1` (parsed as a float).

Reply:
Thanks, will give that a try!